webanno-ui-project/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/project/ProjectUsersPanel.java

```diff
@@ -377,6 +404,7 @@ public UserDetailForm(String id)
                 @Override
                 public void onSubmit()
                 {
+                	filterText.setModel(Model.of(""));
```

Comment:
Instead of changing the model, you should change the value of the variable that the widget is bound to. Here that would be a "userfilter" field in the SelectionModel class. The variable should be called `userFilter` with capital `F` according to CamelCase.